spec/features/projects/projects_index_spec.rb

```diff
@@ -732,28 +732,51 @@ def expect_projects_in_order(*projects)
 
       pending "NOT WORKING YET: Date vs. DateTime issue: Selecting same date for from and to value shows projects of that date"
     end
+
+    feature 'public filter' do
+      scenario 'filter on "public" status' do
+        load_and_open_filters admin
+
+        expect(page).to have_text(project.name)
+        expect(page).to have_text(public_project.name)
+
+        SeleniumHubWaiter.wait
+        projects_page.filter_by_public('no')
```

Comment:
Would it make sense to include those `SeleniumHubWaiter` statements into the `filter_by_public` method?